### PR TITLE
Release/1.3.14

### PR DESCRIPTION
**Pull request by xaun**
@cesar-tonnoir please review, thanks mate :beers:

### CODE REVIEW

File: src/services/utilities/utilities.svc.coffee [hunk 1]
```diff
@@ -19,15 +20,38 @@ angular
       maxDate = _.max _.map(dates, (d) -> moment(d))
       return [minDate.startOf('day').toDate(), maxDate.startOf('day').toDate()]
 
-    getPeriodWord = (period) ->
+    ###
```
Comment (cesar-tonnoir) [inline src/services/utilities/utilities.svc.coffee]: automated doc? :)

**Comment by cesar-tonnoir** [comment]
:cocktail: LGTM - cool to have added helpers to Utilities.svc

File: src/services/utilities/utilities.svc.coffee (hunk 1 above)
Comment (xaun) [inline src/services/utilities/utilities.svc.coffee]: @cesar-tonnoir not auto, but Im going to make it a convention to ensure doc on public methods following this conventions: http://usejsdoc.org/

Comment (cesar-tonnoir) [inline src/services/utilities/utilities.svc.coffee]: :+1: 
don't forget the changelog as well

**Pull request merged**